<launch>
	
	<rosparam command="load" file="$(find jaco_learning)/config/path_follower.yaml" />
	
	<!-- Trajopt PID control following a planned trajectory. -->
	<node name="path_follower" pkg="jaco_learning" type="path_follower.py" output="screen"/>

</launch>

